### ISSUE TITLE
Infos und Hinweise bitte hier...

### ISSUE THREAD

**Issue by dominikbraun** [dominikbraun/WARUM-IST-DAS-CODING-BOARD-OFFLINE#1]

**Comment by asccc** [comment]
Frage ich mich auch, Lano weiß auch nix :(

**Comment by jast** [comment]
Nein, ich weiß auch nichts dazu...

**Comment by dominikbraun** [comment]
Da nun einige wichtige Mitglieder wissen, dass es dieses Repo als Austauschmöglichkeit gibt, habe ich einen weniger reißerischen Namen gewählt.

@jast: Danke schonmal 👍

**Comment by lord-haffi** [comment]
Ja, würde mich auch interessieren. Was ist eigentlich mit dem Serveradmin? Den hat man (/ich) schon ewig nicht mehr auf'm Board gesehen. Nicht, dass er uns so unerwartet wie -AB- vorzeitig verlassen hat...

**Comment by Sven-Moennich** [comment]
Ruhig bleiben. Wenn schon Spekulation dann bitte das da einfach ne Rechnung übersehen wurde. Umbaumaßnahmen wären angekündigt. Tod wäre ein einstellen des Vertrages, da ändert man nicht noch die index.html ohne Nachricht. Wir hatten das doch schon mal ein paar Tage. So lang da nicht jeder mal 5€ aufn Tisch legt kann das halt passieren. Ruhig bleiben, abwarten. Nicht jeden hält ne schottische Deckenleuchte.

**Comment by lord-haffi** [comment]
Ach, so richtig ernst gemeint war das jetzt auch nicht ^^' Grundsätzlich habe ich aber auch nichts dagegen, wenn der jute Admin sich die Page mal durch Spenden oder wenigstens Werbung finanzieren lassen würde... Aber gut, das hat jetzt eher wenig mit diesem Thema zu tun. Jetzt heißt's wohl erstmal Abwarten und Tee trinken.

**Comment by asccc** [comment]
Naja, sollte das Forum wirklich aus Zeit/Kostengründen eingestellt worden sein, wäre zumindest eine kurze Ankündigung nett gewesen. Wäre schade um die Ressourcen und Tutorials der Mitglieder.

**Comment by german-one** [comment]
Ältere Inhalte lassen sich ggf. immer noch abrufen.
https://web.archive.org/web/20181127175607/https://www.coding-board.de/

**Comment by dominikbraun** [comment]
Gibt es inzwischen etwas Neues?

**Comment by jast** [comment]
Ich hab noch nichts gehört.

**Comment by dominikbraun** [comment]
Meine Hoffnungen, dass das CB wieder online geht, schwinden langsam. 👀

**Comment by jr-cologne** [comment]
Allerdings, so langsam mache ich mir auch Sorgen...

Hat denn irgendjemand in den letzten paar Monaten überhaupt noch was von Maik gehört?

**Comment by Sven-Moennich** [comment]
Hat wer bei Xenforo einen Account? Da muss man sich erst Hochposten damit man ne Nachricht schreiben kann.

**Comment by jr-cologne** [comment]
@Sven-Moennich Was meinst du bzw. was hast du vor?

**Comment by Sven-Moennich** [comment]
@jr-cologne Maik war da zwar auch schon länger nicht mehr aber ggf erreicht ihn so eine Nachricht? Hatte gehofft das @jast ne email hätte oder so.

**Comment by jr-cologne** [comment]
Ach, so. Die E-Mail-Adresse zur Kontaktierung des coding-boards ist ja kein Geheimnis. Denke, dass @jast versucht, darüber Kontakt aufzunehmen.

**Comment by Sven-Moennich** [comment]
Gehn wir mal davon aus das die mails des coding-board auch tot sind. Ich hatte gehofft das es da noch ne andere gibt.

**Comment by jast** [comment]
Ja, ich kenne eine andere Adresse von Maik - aber ich habe bis jetzt keine Antwort. Ich vermute, dass Maik gerade irgendwie im Stress steckt oder so - das ist aber ins Blaue geraten, weil wir nie nennenswert Kontakt außerhalb des CB hatten.

Ist zwar schade, dass wir im Moment nichts Genaues wissen, aber er antwortet halt, wenn er antwortet. Wenn ich was höre, sage ich euch natürlich Bescheid.

**Comment by asccc** [comment]
> Stand 11.12.2019
> coding-board bleibt bis auf weiteres offline.
> Ob, oder wann coding-borad wieder online geht ist noch unklar. 

... der Grund wäre interessant gewesen, aber naja.
Das war's dann wohl.

**Comment by jr-cologne** [comment]
Hm, mal abwarten, ob Maik @jast noch eine ordentliche Antwort gibt...

Ich würde ja vorschlagen, wir versuchen Maik zu überzeugen, dass wir das coding-board übernehmen können.

**Comment by asccc** [comment]
Generell ist das keine schlechte Idee, nur befürchte ich, dass der Grund für die eilige Abschaltung der Seite nicht trivial ist. Vielleicht gibt es rechtliche Probleme (DSGVO lässt Grüßen).

Ob man diese Plattform dann übernehmen will ... schwierig.

Solange Maik die Situation nicht aufklärt (zumindest den Stammusern gegenüber) halte ich mich da mal lieber bedeckt. Vielleicht handelt es sich auch um persönliche Gründe, dann geht es uns sowieso nichts an und wir müssen uns damit eben abfinden.

**Comment by jr-cologne** [comment]
Mag sein, dass es einen rechtlichen Grund gibt, warum Maik sich dazu entschieden hat, das Forum offline zu nehmen.

Fände es nur extrem schade, wenn es letztlich an rechtlichen Problemen scheitern sollte. Ist ja schließlich nichts, was sich vielleicht nicht irgendwie doch lösen lässt und dementsprechend zwingend eine Abschaltung nach sich zieht.

Insbesondere wenn die Abschaltung persönliche Gründe hätte, fände ich es ebenso schade, wenn man nicht versucht, anderweitig eine Lösung zu finden, um den Betrieb aufrecht zu erhalten.

Versteht mich nicht falsch: Das ist natürlich zu 100% Maiks Angelegenheit. Da müssen wir gar nicht drüber diskutieren.
Das heißt ja aber nicht, dass man, vorausgesetzt er sieht sich nicht mehr in der Lage das coding-board zu betreiben, nicht freundlich anfragen kann, ob er das coding-board ggf. abtreten würde, um die Community zu erhalten.
Bekommt @jast jetzt keine Antwort, kann man das allerdings ohnehin vergessen.

Nun, ja. Wie gesagt, abwarten und Tee trinken.
Hoffentlich sind die Gründe der Abschaltung nicht allzu schwerwiegend und es gibt noch Hoffnung auf eine Fortsetzung des coding-boards.

**Comment by BAGZZlash** [comment]
Hab mir mal extra für diesen Chat nen Account erstellt... :-)

**Comment by dominikbraun** [comment]
> Hab mir mal extra für diesen Chat nen Account erstellt... :-)

Super, wie hast du uns denn gefunden? :smile:

**Comment by BAGZZlash** [comment]
Google.

**Comment by IngressoDev** [comment]
Ich glaube wir müssen uns damit abfinden, dass das Board nicht wieder kommt. Von daher schlage ich vor eine neue Community zu gründen. Eine wirkliche Alternative scheint es nämlich nicht zu geben.
Ist halt nur sehr schade um den Content.

**Comment by dominikbraun** [comment]
Tja, aber wer soll das betreiben? :eyes:

**Comment by jr-cologne** [comment]
Ich hatte mich privat über Twitter auch schon mal mit @Sven-Moennich darüber unterhalten.
Es wird zunehmend unwahrscheinlicher, dass das coding-board je samt Content wieder online geht.
Sei es mit oder ohne Maik als Betreiber.

Ich könnte mir von daher durchaus vorstellen, ein neues Forum ins Leben zu rufen und nochmal von Grund auf neu zu beginnen. Eine Hand voll aktive Mitglieder hätten wir ja vielleicht schon.
Die anfängliche Leere aufgrund des fehlenden Contents ist natürlich schade, aber irgendwann musste bekanntlich ja jeder so starten.

Wenn niemand etwas dagegen hätte, würde ich mich als Betreiber des Forums anbieten.
Es wäre mit Sicherheit eine tolle Erfahrung sowas voranzutreiben, allerdings würde ich das Ganze mehr als gemeinsames Community-Projekt gestalten.
Wir hatten ja schon im alten coding-board die Idee, gemeinsam Projekte anzugehen und das Forum zu verbessern.
Warum ändern wir unsere Zielsetzung nicht ein bisschen, bauen gemeinsam ein neues Forum auf und setzen die Idee des Community-Projekts endlich in die Tat um?
Bisher ist es dazu ja leider nicht gekommen...

**Comment by german-one** [comment]
Ich bin Web Dev Legastheniker. Da kann ich nicht viel beitragen. Aber von so ziemlich jeder Ressource die ich mal geschrieben hatte hab ich noch ein Backup rumfliegen das ich wieder beisteuern kann ...

**Comment by dominikbraun** [comment]
> allerdings würde ich das Ganze mehr als gemeinsames Community-Projekt gestalten.

Was hättest du denn konkret vor?

**Comment by BAGZZlash** [comment]
Ich bin von der Idee auch begeistert, wüsste aber auch nicht so recht, wie ich mich einbringen könnte. Wichtig ist, dass @german die Ressource mit den Essensresten verlinkbar postet! :-)

**Comment by jr-cologne** [comment]
> > allerdings würde ich das Ganze mehr als gemeinsames Community-Projekt gestalten.
> 
> Was hättest du denn konkret vor?

Damit meine ich erst einmal, dass jeder die Möglichkeit haben sollte, die Entwicklung des Forums mitzugestalten, aktiv Vorschläge zur Verbesserung einzubringen und letztlich diese auch (gemeinsam) umzusetzen.

Wir hatten ja im coding-board schon manche Dinge angesprochen, wo es Ideen und Vorschläge gab, wie man das Forum verbessern könnte.
Dadurch dass man durch die Forensoftware relativ eingeschränkt ist und diese in der Regel nicht gerade günstig daherkommt, könnte man die Forensoftware ja auch selbst entwickeln bzw. auf einem anderen Open-Source-System aufbauend weiterentwickeln. :smile: 

Ebenso wäre es wohl sinnvoll, das Forum im Vergleich zum coding-board ein Stück weit zu modernisieren, vielleicht ein paar neue Themenschwerpunkte zu setzen und entsprechend alles ein wenig umzustrukturieren.

Ansonsten habe ich aber noch keine wirklich konkreten Vorstellungen. Das unterliegt dann zumindest teilweise auch der gemeinsamen Ausgestaltung sowie den Vorstellungen, auf die man sich so einigt.

Um meine Beweggründe zu meinem Angebot vielleicht nochmal ein bisschen besser zu erklären:
Hätte Maik sich auf @jast's Mail gemeldet, hätte ich, vorausgesetzt Maik gibt das coding-board auf und möchte es ggf. übergeben, in Betracht gezogen, das Forum zu kaufen und so das coding-board mehr oder weniger in der aktuellen Form weiterzuführen.
Sprich: Ich würde im Hintergrund die technischen Pfäden zum Fortbetrieb ziehen und gleichzeitig versuchen, das Forum ein bisschen zu modernisieren/aktivieren und das Community-Projekt zu etablieren.
Ansonsten hätte sich aber nicht allzu viel geändert.

Da das allerdings wohl nicht infrage kommt, wenn das coding-board jetzt noch länger offline bleibt, und wenn es eine entsprechende Nachfrage danach gibt, würde ich tatsächlich überlegen, ein neues Forum aufzuziehen.
Eine wirkliche Alternative zum coding-board ist mir nämlich auch noch nicht untergekommen und so ganz ohne Coding Community ist es mir dann ein bisschen still und einsam. :smile: 
Dadurch, dass wir ja prinzipiell schon eine sehr nette und engagierte Truppe im coding-board hatten, würde uns gemeinsam schon was Gutes einfallen, um dem neuen Forum schnell Aufwind zu verleihen.
Und wenn nicht? Auch nicht dramatisch. Dann nutzt man das Forum eben für den Austausch innerhalb eines deutlich kleineren Kreises und hat ggf. auch so viel Spaß.

Ich würde mich dementsprechend bei einem möglichen neuen Forum als Betreiber anbieten, mich um die technischen Angelegenheiten und den grundlegenden Betrieb des Forums kümmern und den rechtlichen Verantwortlichen spielen.
Gleichzeitig bin ich mir aber auch bewusst, dass es eben alleine nicht funktionieren wird und ich auch nicht die Zeit haben werde, immer alles zu regeln.
Aus diesem Grunde bin ich da dann auch auf eure Unterstützung und Mitgestaltung angewiesen.

Kurzum: Ich möchte jetzt nicht einfach so drauflos für mich entscheiden, eine Art neues coding-board aufzumachen, wenn eh kaum noch jemand Interesse daran hat und die Community sowieso nicht mehr fortbestehen kann.
Es ist also nur ein Angebot für den Fall, dass keiner so richtig selbst Initiative ergreifen und ein neues Forum gründen möchte, aber das Interesse und die Begeisterung für das Forum trotzdem noch vorhanden ist.

Gut, ich hoffe, das dient erstmal dem besseren Verständnis meines Vorschlags.

**Comment by dominikbraun** [comment]
Da ich weiß, wie gerne (und ausführlich) hier diskutiert wird, habe ich es mir mal erlaubt, ein neues Issue dafür anzulegen:

[Neues Forum? Anregungen, Ideen, Umsetzung](https://github.com/dominikbraun/coding-board-community/issues/2)

Auf jeden Fall schonmal meinen Dank an alle, die sich bisher beteiligt haben :+1:

**Comment by german-one** [comment]
(side note @BAGZZlash  - the Ukrainian developer you marked will certainly visit this issue but I wonder if he has the slightest idea what "Essensreste" you were talking about 😜)

**Comment by BAGZZlash** [comment]
Verdammt... ;-)

**Comment by Mercury1987** [comment]
Leider ist es jetzt traurige Gewissheit , coding-board.de bleibt offline :-(
`Hallo zusammen,
leider muss ich euch mitteilen, dass das coding-board offline bleiben wird.
Ich musste das Forum unangekündigt offline nehmen, weil es kleinere technische Probleme gab, ich aber wenig Zeit hatte mich darum zu kümmern.
Der eigentliche Grund warum das Forum offline bleibt, ist dass ich in den letzten Monaten mehrmals Mails bekommen habe mit der Drohung mich wegen irgendwelcher Datenschutz Angelegenheiten bei den zuständigen Behörden „anzuscheißen“.
Ich habe nicht die Zeit mich mit diesem Thema intensiv zu beschäftigen.
Unter diesen Voraussetzungen sehe ich für mich als Privatperson keine Möglichkeit mehr ein Internetforum zu betreiben.
Ich sehe leider auch keine Möglichkeit das Forum an jemanden zu übergeben, ohne wieder mit dem Datenschutz in Konflikt zu kommen.
Danke an alle User und ans Team für die vielen Jahre.
Maik `
Werd mich eurem neuen Forum anschließen, da mir das coding-board sehr am Herzen lag. Hab durch langes Suchen bei Google diesen Issue gefunden und freu mich, dass sich einige gefunden haben, die weitermachen wollen :-) 
Gruß
Christian (PosChris bei CB 👍 )

**Comment by dominikbraun** [comment]
Wunderbar, da hat sich wohl noch jemand gefunden @jr-cologne :)

**Comment by jr-cologne** [comment]
@Mercury1987 Schön, dass du zu uns gefunden hast. Ich lade dich in unseren Projektkreis für das neue Forum ein. So bleibst du auf dem Laufenden und kannst dich ggf. beteiligen, wenn du dazu Lust hast.
